tools/testbench/panic.c

```diff
@@ -6,4 +6,8 @@
 
 #include <sof/debug/panic.h>
 
-void __panic(uint32_t p, char *filename, uint32_t linenum) { }
+void __panic(uint32_t p, char *filename, uint32_t linenum)
+{
+	while (1)
+		;
+}
```

Comment:
I'm not familiar with the testbench so not sure what to do here. @jajanusz would  some `while(1) sleep` be better?

Reply:
something like `abort()` will be better, cos at least program will crash and it's easier to debug than freeze

Reply:
Also tools should be more clever when we switch eventually to c11, cos there is `_Noreturn void abort(void);`

Reply:
@marc-hb It should be separate commit - before adding attributes

Reply:
Github shows that it's 2nd commit. I preferred it to be the 1st, cos It may break git bisect for testbench if you have noreturn attribute for function without abort in it, but I'm not sure.

Reply:
The addition of the `abort()` is the first commit. It's hard to tell because commits are ordered sometimes chronologically and sometimes in reverse order and because 
[github doesn't care about git history](https://github.com/isaacs/github/issues/959#issuecomment-593131237) (vendor lock-in), but if you click on this link and check the "parent" you can make sure: https://github.com/thesofproject/sof/commit/a343042b959dfbb

Reply:
> Also tools should be more clever when we switch eventually to c11,

https://stackoverflow.com/questions/48615184/does-visual-studio-2017-fully-support-c99

(no)

Reply:
You're right, I forgot that GH doesn't care about order and sorts by time ...